youths/models.py

```diff
@@ -0,0 +1,61 @@
+import reversion
+from django import forms
+from django.conf import settings
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.forms import SelectMultiple
+
+from profiles.models import Profile
+
+
+class ChoiceArrayField(ArrayField):
+    """
+    A field that allows us to store an array of choices.
+    Uses Django's Postgres ArrayField and a MultipleChoiceField for its formfield.
+    Adapted from https://gist.github.com/danni/f55c4ce19598b2b345ef
+    """
+
+    def formfield(self, **kwargs):
+        defaults = {
+            "form_class": forms.MultipleChoiceField,
+            "choices": self.base_field.choices,
+            "widget": SelectMultiple,
+        }
+        defaults.update(kwargs)
+        return super(ArrayField, self).formfield(**defaults)
+
+
+@reversion.register()
+class YouthProfile(models.Model):
+    # Required info
+    profile = models.OneToOneField(Profile, on_delete=models.CASCADE)
+    ssn = models.CharField(max_length=11)  # TODO ssn validation?
+    school_name = models.CharField(max_length=128)
+    school_class = models.CharField(max_length=10)
+    expiration = models.DateField(null=True, blank=True)
+
+    # Optional info
+    preferred_language = models.CharField(
+        max_length=32, choices=settings.LANGUAGES, default=settings.LANGUAGES[0][0]
+    )
+    volunteer_info = models.TextField(null=True, blank=True)
```

Comment:
please, avoid using `null=True` along with `blank=True` on `TextField` and other types of `CharField`. [this comment on SO](https://stackoverflow.com/a/52217122/7219196) gives the shortest explanation, why you should do it that way, other answer on that SO page give more detailed explanations. 🙂

Reply:
TLDR just use `blank=True` on you `CharField`s to avoid hiccups in the future

Reply:
Will do. Many thanks for the heads up